src/features/dashboard/components/columns.tsx

```diff
@@ -0,0 +1,160 @@
+import { ColumnDef } from '@tanstack/react-table';
+import { TableColumnHeader } from '@/components/TableColumnHeader';
+
+export type StockTableData = {
```

Comment:
Ideally this type should line in a sibling types.ts file. You can make a folder named `columns` then have the `columns.tsx` and `types.ts` inside